src/engagement_db_to_analysis/configuration.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+
+
+@dataclass()
+class DatasetTypes:
+    DEMOGRAPHIC = "demographic"
+    RESEARCH_QUESTION_ANSWER = "research_question_answer"
+
+
+@dataclass
+class AnalysisConfiguration:
```

Comment:
I think this is an AnalysisDatasetConfiguration rather than an AnalysisConfiguration, as it configures one dataset rather than a full analysis.

Reply:
updated thanks!